### ISSUE TITLE
res.send is not a function

### ISSUE THREAD

**Issue by kmelve** [WilliamDASILVA/nuxt-robots-module#2]
Followed the installation instructions and got this message:

```js
this.addServerMiddleware({
    path: 'robots.txt',
    handler(req, res) {
      res.setHeader('Content-Type', 'text/plain');
      res.send(renderedRobots);
    },
  });
};
```

I'm on `"nuxt": "^1.0.0-rc11"`

**Comment by nbau21** [comment]
I am getting the same issue, same version of nuxt

**Comment by WilliamDASILVA** [comment]
Fixed in 1.2.0
